### PR TITLE
[FAB-17290] Change Nexus URL's to point to Artifactory

### PR DESCRIPTION
**Pull request by lindluni**
This change points the nexus URL's to Artifactory artifacts

I will open a Jira issue and add FAB numbers later

Signed-off-by: Brett Logan <Brett.T.Logan@ibm.com>

### CODE REVIEW

File: tools/operator/testdata/kafka-leveldb-notls-network-spec.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #! fabricVersion:
 #! Released images are pulled from docker hub hyperledger/, e.g. 1.4.1 or 2.0.0
 #! Development stream images are pulled from
-#! nexus3.hyperledger.org:10001/hyperledger/, e.g. 1.4.1-stable or 2.0.0-stable
+#! hyperledger-fabric.jfrog.io/, e.g. 1.4.1-stable or 2.0.0-stable
 fabricVersion: 2.0.0-stable
```
Comment (rameshthoomu) [inline tools/operator/testdata/kafka-leveldb-notls-network-spec.yml:9]: It seems we are publishing images with "latest" tag. We also have similar files in release-1.4 branch. Publishing with "latest" tag in 2.0 branch and 1.4.5-stable for 1.4 branch is fine.

File: tools/operator/launcher/networkspec.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 #! fabricVersion:
 #! Released images are pulled from docker hub hyperledger/, e.g. 1.4.1 or 2.0.0
 #! Development stream images are pulled from
-#! nexus3.hyperledger.org:10001/hyperledger/, e.g. 1.4.1-stable or 2.0.0-stable
+#! hyperledger-fabric.jfrog.io/, e.g. 1.4.1-stable or 2.0.0-stable
 fabricVersion: 1.4.5-stable
```
Comment (rameshthoomu) [inline tools/operator/launcher/networkspec.yaml:9]: Are we publishing images with this tag?

File: tools/operator/templates/k8s/fabric-k8s-pods.yaml [hunk 3]
```diff
@@ -62,7 +62,7 @@
 #@ def caContainers(input, orgName, config):
 #@   ca_image = "hyperledger/fabric-ca:{}".format(config.fabricVersion)
 #@   if config.fabricVersion.endswith("-stable"):
-#@     ca_image="nexus3.hyperledger.org:10001/hyperledger/fabric-ca:amd64-{}".format(config.fabricVersion)
+#@     ca_image="hyperledger-fabric.jfrog.io/hyperledger/fabric-ca:amd64-{}".format(config.fabricVersion)
```
Comment (rameshthoomu) [inline tools/operator/templates/k8s/fabric-k8s-pods.yaml]: remove "hyperledger" org name after jfrog.io url

Comment (lindluni) [reply tools/operator/templates/k8s/fabric-k8s-pods.yaml]: Good catch

File: tools/operator/launcher/networkspec.yaml (hunk 2 above)
Comment (lindluni) [reply tools/operator/launcher/networkspec.yaml:9]: Which tag?

File: tools/operator/testdata/kafka-leveldb-notls-network-spec.yml (hunk 1 above)
Comment (lindluni) [reply tools/operator/testdata/kafka-leveldb-notls-network-spec.yml:9]: Correct, I need to do this work in 1.4 as well. We will publish stable and latest in master, and only stable in 1.4

File: tools/operator/launcher/networkspec.yaml (hunk 2 above)
Comment (rameshthoomu) [reply tools/operator/launcher/networkspec.yaml:9]: 1.4.5-stable?

File: tools/operator/testdata/kafka-leveldb-notls-network-spec.yml (hunk 1 above)
Comment (scottz64) [reply tools/operator/testdata/kafka-leveldb-notls-network-spec.yml:9]: in master, is latest for the dev branch (post-merge), and, the 2.0.0-stable for the post-interop integratioin branch?

**Comment by rameshthoomu** [comment]
LGTM

File: tools/operator/testdata/kafka-leveldb-notls-network-spec.yml (hunk 1 above)
Comment (lindluni) [reply tools/operator/testdata/kafka-leveldb-notls-network-spec.yml:9]: @scottz64 This doesn't bring any of the interop methodology into play yet, this just replicates what we have today

File: tools/operator/launcher/networkspec.yaml (hunk 2 above)
Comment (lindluni) [reply tools/operator/launcher/networkspec.yaml:9]: I'll update the example to say 1.4.5-stable, but that was just an example

File: scripts/pullDockerImages.sh [hunk 4]
```diff
@@ -34,31 +34,31 @@ dockerTag() {
...
-    if ! docker pull $ORG_NAME-$IMAGE:latest > /dev/null; then
+    echo "$ORG_NAME/fabric-$IMAGE"
+    if ! docker pull $ORG_NAME/fabric-$IMAGE:latest > /dev/null; then
        echo  "FAILED: Docker Pull Failed on $IMAGE"
        exit 1
     fi
-    docker tag $ORG_NAME-$IMAGE:latest $ORG_NAME-$IMAGE:$ARCH-latest
+    docker tag $ORG_NAME/fabric-$IMAGE:latest $ORG_NAME/fabric-$IMAGE:$ARCH-latest
   done
 }
```
Comment (rameshthoomu) [inline scripts/pullDockerImages.sh:63]: In fabric master branch, we are pulling these images from it's source repos. See here https://github.com/hyperledger/fabric/blob/master/Makefile#L153-L157. Update the same here.

Comment (rameshthoomu) [reply scripts/pullDockerImages.sh:63]: Created JIRA for this task FAB-17295. Will address this comment in next patch.

**Review by scottz64 — CHANGES_REQUESTED** [review]
1. I am confused why we use a different link for the binaries than for the images. Nevertheless, this has been tested successfully by Bharadwaj so that confusion is mine. If you can explain, that would be helpful.

2. I think this is ready to merge - except that we should wait until @btl5037 automates his script to push images and binaries to artifactory; right now there are artifacts to pull, but they are not being updated daily like the Nexus images are updated. Thus, without that being automated, our daily tests would always pull those same images (dated from a couple weeks ago).

**Review by scottz64 — APPROVED** [review]

**Pull request merged**